src/agent/onefuzz/src/asan.rs

```diff
@@ -136,8 +136,8 @@ pub async fn check_asan_string(mut data: String) -> Result<Option<AsanLog>> {
     if asan.is_some() {
         return Ok(asan);
     } else {
-        if data.len() > 1024 {
-            data.truncate(1024);
+        if data.len() > 4096 {
```

Comment:
Shall we add a constant for this value, if you're changing it anyway?

I'm guessing the change is not necessary, but just made debugging easier and produces a better UX.

Reply:
Added const.

It wasn't specifically needed, but as you mentioned, gives more context for the asan log, which makes it easier to debug.